compiler/ballerina-parser/src/test/resources/statements/match-stmt/match_stmt_source_16.bal

```diff
@@ -0,0 +1,4 @@
+function func(any v){
```

Comment:
Missing license header.

Reply:
Added thanx.

Reply:
We haven't added licence headers to any of the parser test bal-sources. The reason for that was, these parser tests operates at a very granular level, and tests a specific grammar rule in each test-case.

Having the license header means, parser has to parse additional grammar rules as well - then we no longer have that granularity in the test..

Can we stick to the same?

Reply:
reverted

Reply:
+1. Thanks for the info @SupunS